From 3f1a7ba1cffb66ec5ba6bc2c2dff5ae5988bcc94 Mon Sep 17 00:00:00 2001
From: "H. Vetinari" <h.vetinari@gmx.com>
Date: Wed, 2 Mar 2022 13:43:37 +1100
Subject: [PATCH 5/7] expand ABSL_INTERNAL_DLL_TARGETS

This is due to how the logic of absl_internal_dll_contains checks
whether additional static libraries should be built, compare
https://github.com/abseil/abseil-cpp/blob/20211102.0/CMake/AbseilHelpers.cmake#L112-L132
---
 CMake/AbseilDll.cmake | 21 +++++++++++++++++++++
 1 file changed, 21 insertions(+)

diff --git a/CMake/AbseilDll.cmake b/CMake/AbseilDll.cmake
index 2c7f162..aec198e 100644
--- a/CMake/AbseilDll.cmake
+++ b/CMake/AbseilDll.cmake
@@ -366,6 +366,11 @@ set(ABSL_INTERNAL_DLL_TARGETS
   "container_common"
   "container_memory"
   "cord"
+  "cord_internal"
+  "cordz_functions"
+  "cordz_handle"
+  "cordz_info"
+  "cordz_sample_token"
   "core_headers"
   "counting_allocator"
   "debugging"
@@ -377,6 +382,18 @@ set(ABSL_INTERNAL_DLL_TARGETS
   "exponential_biased"
   "failure_signal_handler"
   "fixed_array"
+  "flags"
+  "flags_commandlineflag"
+  "flags_commandlineflag_internal"
+  "flags_config"
+  "flags_internal"
+  "flags_marshalling"
+  "flags_parse"
+  "flags_private_handle_accessor"
+  "flags_program_name"
+  "flags_reflection"
+  "flags_usage"
+  "flags_usage_internal"
   "flat_hash_map"
   "flat_hash_set"
   "function_ref"
@@ -396,6 +413,7 @@ set(ABSL_INTERNAL_DLL_TARGETS
   "leak_check"
   "leak_check_disable"
   "log_severity"
+  "low_level_hash"
   "malloc_internal"
   "memory"
   "meta"
@@ -410,6 +428,7 @@ set(ABSL_INTERNAL_DLL_TARGETS
   "random_bit_gen_ref"
   "random_distributions"
   "random_internal_distribution_caller"
+  "random_internal_distribution_test_util"
   "random_internal_distributions"
   "random_internal_explicit_seed_seq"
   "random_internal_fast_uniform_bits"
@@ -445,6 +464,8 @@ set(ABSL_INTERNAL_DLL_TARGETS
   "stack_consumption"
   "stacktrace"
   "status"
+  "statusor"
+  "strerror"
   "str_format"
   "str_format_internal"
   "strings"
-- 
2.35.1.windows.2

